src/components/Books.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function Books({ title, author }) {
+  return (
+    <div className="main">
+      <div>
+        <h3>
+          Title:
+          {'me '}
+          {title}
+        </h3>
+        <h3>
+          By:
+          {' me'}
+          {author}
+        </h3>
```

Comment:
- [ ] What is needed here is to use the props and the values would be displayed. Something like this :point_down: 

```
   <h3>{title}</h3>
     
   <h3>Byt: {author}</h3>
```